lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
Ok, based on [this benchmark](https://esbench.com/bench/598ff32299634800a0348d94), I'd strongly suggest to go for `F.p.apply` and rest parameters instead, as indicated by the [results](https://twitter.com/bmeurer/status/896623943066562560) for Chrome Canary:

![image](https://user-images.githubusercontent.com/815988/29246797-3107b34c-8004-11e7-8fc3-f3b804f87772.png)